src/main/java/org/dwcj/controls/ComboBox.java

```diff
@@ -20,14 +20,23 @@
 /**
  * Combobox Control
  */
-public final class    ComboBox extends AbstractDwclistControl implements IStyleable, IThemable, IExpansible {
+public final class ComboBox extends AbstractDwclistControl {
 
     private BBjListButton bbjListButton;
 
-    private BBjComboBoxSelectEventSink comboBoxSelectEventSink;
+    private ComboBoxSelectEventSink comboBoxSelectEventSink;
 
     private ComboBoxChangeEventSink comboBoxChangeEventSink;
```

Comment:
what was the outcome of our discussion again? change, not select for the event name? Or both?